### PR TITLE
Dịch "transformer_vn" - Phần 8

### LINKED ISSUE #1759: Dịch "transformer_vn" - Phần 8
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_attention-mechanisms/transformer_vn.md#L737-L821

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by davidnvq**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #1759

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_attention-mechanisms/transformer_vn.md [hunk 1]
```diff
@@ -800,7 +806,11 @@ for sentence in ['Go .', 'Wow !', "I'm OK .", 'I won !']:
 * Positional encoding is the only place that adds positional information to the Transformer model.
 -->
 
-*dịch đoạn phía trên*
+* Mô hình Transformer dựa trên kiến ​​trúc mã hóa - giải mã.
+* Tầng tập trung đa đầu gồm có $h$ tầng tập trung song song.
+* Mạng truyền xuôi theo vị trí gồm hai tầng kết nối đầy đủ được áp dụng trên chiều cuối cùng.
+* Chuẩn hoá theo tầng khác với chuẩn hoá theo batch đó là chuẩn hoá trên chiều cuối cùng (chiều đặc trưng) thay vì chiều đầu tiên (kích thước batch).
+* Biểu diễn vị trí là cách duy nhất đưa vào mô hình Transformer thông tin về vị trí trong chuỗi.
```
Comment (lkhphuc) [inline chapter_attention-mechanisms/transformer_vn.md:813]:
```suggestion
* Biểu diễn vị trí là nơi duy nhất thông tin vị trí trong chuỗi được đưa vào mô hình Transformer.
```
Có cách khác nhưng transformer không dùng.

File: chapter_attention-mechanisms/transformer_vn.md [hunk 2]
```diff
@@ -771,20 +773,24 @@ d2l.train_s2s_ch9(model, train_iter, lr, num_epochs, ctx)
...
+Chúng ta có thể sử dụng Transformer đã được huấn luyện để dịch một số câu đơn giản dưới đây.
+
 
 
 ```{.python .input  n=20}
 for sentence in ['Go .', 'Wow !', "I'm OK .", 'I won !']:
     print(sentence + ' => ' + d2l.predict_s2s_ch9(
         model, sentence, src_vocab, tgt_vocab, num_steps, ctx))
 ```
+Sau đó ta khởi tạo trạng thái của bộ giải mã bằng cách truyền vào ba đầu ra thu được từ bộ mã hoá:
```
[the remark this answers is not in the document]
Comment (davidnvq) [reply chapter_attention-mechanisms/transformer_vn.md:793]: câu này sao Phúc ơi :)) thừa à :3

File: chapter_attention-mechanisms/transformer_vn.md [hunk 3]
```diff
@@ -771,13 +773,16 @@ d2l.train_s2s_ch9(model, train_iter, lr, num_epochs, ctx)
 As we can see from the training time and accuracy, compared with the seq2seq model with attention model, Transformer runs faster per epoch, and converges faster at the beginning.
 -->
 
-*dịch đoạn phía trên*
+Dựa trên thời gian huấn luyện và độ chính xác, ta thấy Transformer chạy nhanh hơn trong mỗi epoch và hội tụ nhanh hơn ở giai đoạn ban đầu so với mô hình seq2seq áp dụng cơ chế tập trung.
+Như chúng ta có thể nhìn thấy từ thời gian huấn luyện và độ chính xác, so với mô hình seq2seq có tập trung, Transformer chạy nhanh hơn trên mỗi epoch và cũng hội tụ nhanh hơn lúc ban đầu.
```
Comment (lkhphuc) [inline chapter_attention-mechanisms/transformer_vn.md:777]:
```suggestion
```
Lặp câu trên rồi.

File: chapter_attention-mechanisms/transformer_vn.md [hunk 4]
```diff
@@ -800,7 +805,11 @@ for sentence in ['Go .', 'Wow !', "I'm OK .", 'I won !']:
 * Positional encoding is the only place that adds positional information to the Transformer model.
 -->
 
-*dịch đoạn phía trên*
+* Mô hình Transformer dựa trên kiến ​​trúc mã hóa - giải mã.
+* Tầng tập trung đa đầu gồm có $h$ tầng tập trung song song.
+* Mạng truyền xuôi theo vị trí gồm hai tầng kết nối đầy đủ được áp dụng trên chiều cuối cùng.
+* Chuẩn hoá theo tầng thực hiên chuẩn hoá trên chiều cuối cùng (chiều đặc trưng),  khác với chuẩn hoá theo batch thực hiện trên chiều đầu tiên (kích thước batch).
```
Comment (lkhphuc) [inline chapter_attention-mechanisms/transformer_vn.md:811]:
```suggestion
* Chuẩn hoá theo tầng thực hiện chuẩn hoá trên chiều cuối cùng (chiều đặc trưng), khác với chuẩn hoá theo batch thực hiện trên chiều đầu tiên (kích thước batch).
```

File: chapter_attention-mechanisms/transformer_vn.md (hunk 3 above)
Comment (davidnvq) [reply chapter_attention-mechanisms/transformer_vn.md:777]: Chac sua lai ma quen chua xoa ;))
